PerfettoProcessor/Events/PerfettoGpuCounterTrackEvent.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using Perfetto.Protos;
+using Utilities;
+
+namespace PerfettoProcessor
+{
+    public class PerfettoGpuCounterTrackEvent : PerfettoSqlEvent
+    {
+        public const string Key = "PerfettoGpuCounterTrackEvent";
+
+        public const string SqlQuery = "select name, id from gpu_counter_track";
```

Comment:
It seems these other fields might be useful and documented here - https://perfetto.dev/docs/analysis/sql-tables#gpu_counter_track

gpu_id, unit, description, source_arg_set_id (for args)

We don't necessarily have to consume them later on